lib/iris/cube.py

```diff
@@ -1835,6 +1835,10 @@ def summary(self, shorten=False, name_padding=35):
 
         nameunit = '{name} / ({units})'.format(name=self.name(),
                                                units=self.units)
+        # If all unknown and a STASH attribute exists, use it.
+        if (nameunit == 'unknown / (unknown)' and
+           'STASH' in self.attributes):
```

Comment:
This line can be joined to the above and it'll still only be 78 characters long.